test/pseudo-tty/test-assert-no-color.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+require('../common');
+const assert = require('assert').strict;
+
+process.env.NODE_DISABLE_COLORS = true;
+
+try {
+  assert.strictEqual(1, 3);
```

Comment:
This won't normally result in colorized output, at least not in my environment. `assert.deepStrictEqual({a: 'a'}, {})` will do the trick.